api/proto/disperser/disperser.proto

```diff
@@ -78,12 +78,12 @@ message AuthenticationData {
 
 message DisperseBlobRequest {
```

Comment:
The other copy: https://github.com/Layr-Labs/eigenda/blob/master/api/proto/disperser/v2/disperser_v2.proto#L25

Reply:
[9d46d92](https://github.com/Layr-Labs/eigenda/pull/780/commits/9d46d9283af20fbf6c2c5dbe48f1cdcc6c6e23e2)